index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="./index.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200&display=swap" rel="stylesheet">
+
+</head>
+  <body>
+    <header>
+     <nav> 
+    <img src="./Images/Hamburger_icon.svg.png" class="hamburger" alt="" />
+      <ul class="menu-items">
+        <img src="./Images/Cross-Icon.png" class="Cross-Icon" alt="" />
+        <li class="menu-item-mobile">
+          <a href="./about.html">About</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Program</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Join</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Sponsor</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Help</a>
+        </li>
+      </ul>
+
+      <div class="myLinks">
+        <ul>
+          <li><a href="www.facebook.com"></a> <img src="/Images/fb.icon.png" alt=""></li>
+          <li><a href="www.gooogle.com"></a><img src="/Images/Twttr-logo.png" alt=""></li>
+          <li><a href="/index.html">Login</a></li>
+          <li><a href="about.html">MyPage</a></li>
+          <li><a href="#">English</a></li>
+        </ul>
+      </div> 
+      
+      <ul class="nav-list">
+        <li class="menu-item">
+          <img src="./Images/My-Logo.png" alt="Logo">
+        </li>
+        <li class="menu-item">
+          <a href="./about.html">About</a>
+        </li>
+        <li class="menu-item">
+          <a href="#">Program</a>
+        </li>
+        <li class="menu-item">
+          <a href="#">Join</a>
+        </li>
+        <li class="menu-item">
+          <a href="#">Sponsor</a>
+        </li>
+        <li class="menu-item">
+        
+        </li>
+      </ul>
+      </nav>
+    </header>
+    <div id="home">
+      <img src="./Images/tiitle.png" alt="">
+      <div id="description">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora
+          facere quae, dolore perspiciatis iste sapiente ut a animi! Voluptate
+        </p>
+      </div>
+      <div id="time">
+        <h2>2020.10.15(THU) ~ 16(FRI)</h2>
+        <h6>@National Museum of Rwanda, Art Center</h6>
+      </div>
```

Comment:
- [ ] As requested by the previous reviewer you should make sure this content is aligned properly to match the design.
  - You can see the changes below
     - The `2020.10.15(THU) ~ 16(FRI)` should be on one line.
     - The `@National Museum of Rwanda, Art Center` should also be one line.
     
| Your Design |Project Design |
| :---:   | :---: | 
| ![Screenshot from 2022-11-26 11-09-15](https://user-images.githubusercontent.com/92303891/204085806-0cd91526-6609-4101-a87b-be49cfa4ecd8.png) | ![image](https://user-images.githubusercontent.com/92303891/204085816-b5d62118-47f5-4afe-bee3-c4f86aba0576.png) |